src/test-utils/dom/app-layout/index.ts

```diff
@@ -63,7 +63,7 @@ export default class AppLayoutWrapper extends ComponentWrapper {
     return this.findByClassName(testutilStyles['drawers-mobile-triggers-container']);
   }
 
-  findDrawersTriggers(): ElementWrapper<HTMLButtonElement>[] | null {
+  findDrawersTriggers(): ElementWrapper<HTMLButtonElement>[] {
```

Comment:
Not directly related to the change, but I found it when writing tests. `findAll` util can never return `null`, so the type can be simplified